.changeset/six-penguins-eat.md

```diff
@@ -0,0 +1,25 @@
+---
+'@sap-cloud-sdk/connectivity': patch
+'@sap-cloud-sdk/eslint-config': patch
+'@sap-cloud-sdk/generator': patch
+'@sap-cloud-sdk/generator-common': patch
+'@sap-cloud-sdk/http-client': patch
+'@sap-cloud-sdk/odata-common': patch
+'@sap-cloud-sdk/odata-v2': patch
+'@sap-cloud-sdk/odata-v4': patch
+'@sap-cloud-sdk/openapi': patch
+'@sap-cloud-sdk/openapi-generator': patch
+'@sap-cloud-sdk/temporal-de-serializers': patch
+'@sap-cloud-sdk/test-util': patch
+'@sap-cloud-sdk/util': patch
+'@sap-cloud-sdk/e2e-tests': patch
+'@sap-cloud-sdk/integration-tests': patch
+'@sap-cloud-sdk/test-services-common': patch
+'@sap-cloud-sdk/test-services-openapi': patch
+'@sap-cloud-sdk/test-services-odata-v2': patch
+'@sap-cloud-sdk/test-services-odata-v4': patch
+'@sap-cloud-sdk/test-services-e2e': patch
+'@sap-cloud-sdk/type-tests': patch
+---
+
+Adjust package.json scripts for testing, generating, and others
```

Comment:
I guess this is the message which will end up in the generate Changelog? For this pr most likely no customer facing information is needed.